lib/gtest/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(gtest_builder C CXX)
 include(ExternalProject)
 
 ExternalProject_Add(gtest
-    URL https://googletest.googlecode.com/files/gtest-1.7.0.zip
+    URL http://pkgs.fedoraproject.org/repo/pkgs/gtest/gtest-1.7.0.zip/2d6ec8ccdf5c46b05ba54a9fd1d130d7/gtest-1.7.0.zip
```

Comment:
I got that from another fork. But I agree that is a better/more reliable link. 
The Travis build is in progress.